modules/RTC/TraceablePeerConnection.js

```diff
@@ -1852,6 +1873,21 @@ TraceablePeerConnection.prototype.setRemoteDescription = function(
         });
     }
 
+    if (this.options.enableLayerSuspension) {
+        logger.debug('Layer suspension enabled,'
+            + `currently selected? ${this.isSelected}`);
+        const bitrateCapKbps = getSuspensionBitrateKbps(this.isSelected);
+
+        this.bandwidthLimiter.setBandwidthLimit('video', bitrateCapKbps);
+        logger.debug(`Layer suspension got bitrate cap of ${bitrateCapKbps}`);
+        description.sdp
+            = this.bandwidthLimiter.enforceBandwithLimit(description.sdp);
+        logger.warn(
```

Comment:
logging on the warning level should happen only if something goes wrong

Reply:
fixed